jekyll/_cci2/workflows.md

```diff
@@ -168,14 +168,14 @@ workflows:
       - test1:
           requires:
             - build
-	  context: org-global  
+    context: org-global  
```

Comment:
Was somehow got replaced with the wrong amount of spaces, making it appear in an odd way.

Fixed in #1536